src/main/java/make_comment/use_case/CommentFactory.java

```diff
@@ -0,0 +1,13 @@
+package make_comment.use_case;
+
+import entities.Comment;
+
+public class CommentFactory {
+
+    //creates and returns a comment
+
+    public static Comment makeComment(int UserId, String body, int id) {
```

Comment:
I don't think makeComment should be static, rather we can pass it in the constructor w/o changing much